process/caller.py

```diff
@@ -111,7 +114,10 @@ def call_models_full_idempotence(self, xc: np.ndarray, ifail: int) -> None:
         # TODO The only way to ensure idempotence in all outputs is by comparing
         # mfiles at this stage
         previous_mfile_arr = None
-        file_prefix = ft.global_variables.fileprefix
+        # Path of output file prefix required for writing intermediate idempotence-
+        # checking mfiles in the same directory as input file
+        output_prefix = ft.global_variables.output_prefix
```

Comment:
Why are we assigning this variable to only use it once? I think it would be less verbose to write

```python
f2py_compatible_to_string(ft.global_variables.output_prefix) + "IDEM_MFILE.DAT"
``` 

on line 131